### ISSUE TITLE
[TF2] Halloween 'Soul' entities still spawn outside of event (Full Moon/Eternaween)

### ISSUE THREAD

**Issue by rabscootle** [ValveSoftware/Source-1-Games#4060]
Hi! I was wondering if there was any chance that the Halloween 'Souls' could be updated to not spawn during Full Moon/Eternaween. They cause a lot of extra sounds and frame drops, and the Soul Gargoyle only increments from these during the annual Halloween season, making them redundant outside of that period.

![image](https://user-images.githubusercontent.com/108029512/179387981-4f436ab7-ab9d-4aef-8b58-609511b48da7.png)

**Comment by aarmastah** [comment]
Alternatively, maybe there could be a client cvar to control whether or not souls appear?

**Comment by agrastiOs** [comment]
Not sure a cvar is needed as this existing outside the Halloween season doesn't seem to be unintentional, as the point of these Soul entities is to telegraph the player that they just received some souls.

**Comment by aarmastah** [comment]
True. I was meaning moreso a cvar that lets you turn them off entirely regardless.
